Gemfile

```diff
@@ -53,6 +53,7 @@ group :development, :test do
   gem 'byebug', platforms: %i(mri mingw x64_mingw)
   gem 'rspec-rails', '~> 4.0.0.beta3'
   gem 'shoulda-matchers'
+  gem 'factory_bot_rails', git: 'https://github.com/thoughtbot/factory_bot_rails.git'
```

Comment:
I guess that git link is not needed, just name of the gem would be ok

Reply:
Had some problems before, before gem was on my machine. Unfortunatelly didn't make any print scr or save of it. I think "bundle update" scream about some strange errors, now when i comment "git: 'http...." part of the entry in Gemfile everything seems to work fine but it wasnt before...